### DISCUSSION TITLE
Google Search for OSS MeshCentral leads to https://meshcommander.com/meshcentral2 with redirect to https://meshcentral.com/info/ as down, 404@github-pages

### DISCUSSION THREAD

**Discussion by classicvalues** [Ylianst/MeshCentral#6086] [Issues]
https://meshcommander.com/meshcentral redirects to https://meshcentral.com/info down as a 404@github-pages and is not https://github.com/Ylianst/MeshCentral/deployments/github-pages where https://ylianst.github.io/MeshCentral/ would be the github-pages site. 

**To Reproduce**
1. Go to https://meshcommander.com/meshcentral as per "MeshCommander" Google Search
2. Click on [The MeshCentral main information website is now here](https://meshcentral.com/info/).
3. Redirects to https://meshcentral.com/info/ as a "404 File not found"
4. Please either fix to https://meshcentral.com/ or https://ylianst.github.io/MeshCentral/

**Expected behavior**
Expected either https://meshcentral.com/ or https://ylianst.github.io/MeshCentral/

**Client Device (please complete the following information):**
 - Device: Laptop
 - OS: Windows
 - Network: Local, Public
 - Browser: Google Chrome

Thank you kindly!

**Comment by si458** [comment]
Thanks for this, i have moved it to a discussion as its not a bug with meshcentral software.
I have reported this issue already to @Ylianst , he just needs to update the meshcommander website as he owns/controls it, so it's a waiting game now 👍
